access reviews to v1 - using doc stub generator (#12207)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/accessreviewqueryscope.md

---
title: "accessReviewQueryScope resource type"

> description: "Defines what will be reviewed in an access review."

Reviewer: ```suggestion
description: "Defines what needs to be reviewed in an access review."
```

author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewQueryScope resource type

Namespace: microsoft.graph

> An accessReviewQueryScope object defines what will be reviewed in an [access review](../resources/accessreviewsv2-root.md). To scope an access review to inactive users, see [accessReviewInactiveUserQueryScope](../resources/accessreviewinactiveusersqueryscope.md). 

Reviewer: ```suggestion
An accessReviewQueryScope object defines what is reviewed in an [access review](../resources/accessreviewsv2-root.md). To scope an access review to inactive users, see [accessReviewInactiveUserQueryScope](../resources/accessreviewinactiveusersqueryscope.md). 
```

---
title: "accessReviewQueryScope resource type"

> description: "Defines what will be reviewed in an access review."

Reviewer 2: ```suggestion
description: "Defines what needs to be reviewed in an access review."
```

author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewQueryScope resource type

Namespace: microsoft.graph

> An accessReviewQueryScope object defines what will be reviewed in an [accessReview](../resources/accessreviewsv2-root.md). See the supported queries to see the selection options. To scope an access review to inactive users, see [accessReviewInactiveUserQueryScope](../resources/accessreviewinactiveusersqueryscope.md).

Reviewer 2: ```suggestion
An accessReviewQueryScope object defines what needs to be reviewed in an [accessReview](../resources/accessreviewsv2-root.md). See the supported queries to see the selection options. To scope an access review to inactive users, see [accessReviewInactiveUserQueryScope](../resources/accessreviewinactiveusersqueryscope.md).
```

Inherits from [accessReviewScope](../resources/accessreviewscope.md).

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |query|String|The query representing what will be reviewed in an access review. Examples of this include /groups/{id}/members?$filter=…|

Reviewer 2: ```suggestion
|query|String|The query that represents what needs to be reviewed in an access review. Examples of this include /groups/{id}/members?$filter=…|
```
